lightly/api/routes/users/docker/service.py

```diff
@@ -1,115 +0,0 @@
-""" Docker Service """
```

Comment:
I know you like to abolish stuff :D But I would keep the docker routes around until we move them to v1. Otherwise we can never upgrade the pip version in the docker.